README.md

```diff
@@ -184,6 +203,38 @@ Keeps the full path of downloaded files from a Google Cloud Storage bucket. Requ
 ### resultsDir
 The name of a unique Google Cloud Storage object within the results bucket where raw test results will be stored. The default is a timestamp with a random suffix.
 
+### runTimeout
+The max time this test run can execute before it is cancelled. s (seconds), m (minutes), h (hours) suffixes are acceptable (default: unlimited).
+
+### ignoreFailedTests
+Terminate with exit code 0 when there are failed tests. Useful for Fladle and other gradle plugins that don't expect the process to have a non-zero exit code. The JUnit XML is used to determine failure. (default: false)
+
+### disableSharding
+Disables sharding. Useful for parameterized tests. (default: false)
+
+### smartFlankDisableUpload
+Disables smart flank JUnit XML uploading. Useful for preventing timing data from being updated. (default: false)
+
+### testRunnerClass
+The fully-qualified Java class name of the instrumentation test runner (default: the last name extracted from the APK manifest).
+
+### localResultsDir
+Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
+
+### numUniformShards
+Specifies the number of shards into which you want to evenly distribute test cases. The shards are run in parallel on separate devices. For example, if your test execution contains 20 test cases and you specify four shards, each shard executes five test cases. The number of shards should be less than the total number of test cases. The number of shards specified must be >= 1 and <= 50. This option cannot be used along max-test-shards and is not compatible with smart sharding. If you want to take benefits of smart sharding use max-test-shards instead. (default: null)
+
+### clientDetails
+A key-value map of additional details to attach to the test matrix. Arbitrary key-value pairs may be attached to a test matrix to provide additional context about the tests being run. When consuming the test results, such as in Cloud Functions or a CI system, these details can add additional context such as a link to the corresponding pull request.
```

Comment:
Can we link to gcloud docs here to better understand how these are passed to the device?
Also an example of how to consume these client details? Are these to be consumed on the test device or in scripts?